### PR TITLE
Orvibo Dry contact and ST30 sensor

### PR DESCRIPTION
**Pull request by Smanar**
Temperature snesor, working fine https://github.com/dresden-elektronik/deconz-rest-plugin/issues/3885
Dry contact https://github.com/dresden-elektronik/deconz-rest-plugin/issues/3973 working fine but model id and manufacture name are empty.

### CODE REVIEW

**Comment by manup** [comment]
>Dry contact #3973 working fine but model id and manufacture name are empty.

Is this empty values issue figured out yet? I think the PR shouldn't be merged before that, since most clients somehow expect these values to be present.

**Comment by manup** [comment]
Any news here regarding the empty values?

**Comment by Smanar** [comment]
Yes, It seem working from the user, I m asking the JSON to check value.

**Comment by Smanar** [comment]
It s ok, the missing value are ok now.
Have just corrected the "model id" correction.

File: de_web_plugin.cpp [hunk 1]
```diff
@@ -5252,6 +5277,10 @@ void DeRestPluginPrivate::addSensorNode(const deCONZ::Node *node, const deCONZ::
                     {
                        // use manufacturer specific cluster instead
                     }
+                    // Don't create sensor for first endpoint
+                    else if ((modelId == QLatin1String("ST30 Temperature Sensor")) && (i->endpoint() != 0x02))
```
Comment (manup) [inline de_web_plugin.cpp:5281]: `else if (modelId == QLatin1String("ST30 Temperature Sensor") && i->endpoint() != 0x02)`

Parentheses...

**Review by manup — COMMENTED** [review]
One small thing otherwise good to go.

In future we should check how to use the `R_GetProductId()` for Orvibo devices to improve these weird model ids in a controlled way.

File: de_web_plugin.cpp (hunk 1 above)
Comment (Smanar) [reply de_web_plugin.cpp]: done

**Pull request merged**